lib/metrics/RedisClient.js

```diff
@@ -22,6 +22,36 @@ class RedisClient {
         return this;
     }
 
+    /**
+    * scan a pattern and return matching keys
+    * @param {string} pattern - string pattern to match with all existing keys
+    * @param {number} [count] - optional param, scan count
+    * @param {callback} cb - callback (error, result)
+    * @return {undefined}
+    */
+    scan(pattern, count, cb) {
+        let callback;
+        // set default count to 10
+        const params = { match: pattern, count: 10 };
+        const keys = [];
+
+        if (!cb) {
+            callback = count;
```

Comment:
I guess since the `count` param is optional, this could actually end up being the callback function? Wondering if there's a more clear way of handling this case rather than a reassignment.

Reply:
For this, I ran into a similar issue in the past, actually it was for:
https://github.com/scality/Arsenal/pull/434/files#diff-9c8ee1136532e66fc124018674bec5e0R79
I think this might've been the simplest solution without breaking anything that relied on it.

That said, this use here is slightly different, but still has a `cb` as a mandatory argument. I could set a default value on count, and call this function passing a null argument maybe?